pkg/types/types.go

```diff
@@ -36,21 +35,24 @@ func PlatformFromEnum(s string) string {
 }
 
 var (
-	referenceFormat = regexp.MustCompile(`^([\w\d-\._]+):([\w\d-\._]+)/?([\w\d-\._]+)?$`)
+	referenceFormat = regexp.MustCompile(`^([^:]+\:)?([\w\d-\._]+)/?([\w\d-\._]+)?$`)
 )
 
 // ParseReference parses a given text as a Reference.
 func ParseReference(text string) (*Reference, error) {
 	matches := referenceFormat.FindStringSubmatch(text)
-	if len(matches) != 4 {
-		return nil, fmt.Errorf("%q is not a valid GetEnvoy reference. Expected format: <flavor>:<version>[/<platform>]", text)
+	if len(matches) == 4 { // only allow historical naming convention when the "flavor" is "standard"
+		flavor := strings.ToLower(matches[1])
+		if flavor == "" || flavor == "standard:" {
+			return &Reference{strings.ToLower(matches[2]), PlatformFromEnum(matches[3])}, nil
+		}
 	}
-	return &Reference{strings.ToLower(matches[1]), strings.ToLower(matches[2]), PlatformFromEnum(matches[3])}, nil
+	return nil, fmt.Errorf("%q is not a valid GetEnvoy reference. Expected format: <version>[/<platform>]", text)
```

Comment:
not splitting hairs here.. if you either use a non-standard old-school flavor or mess up the syntax, we err. To be compat with old references, tolerating "standard:1.17.1" style syntax even if we don't care about the standard part